db/migrate/20160119092239_add_show_filter_to_custom_fields.rb

```diff
@@ -0,0 +1,6 @@
+class AddShowFilterToCustomFields < ActiveRecord::Migration
+  def change
+    add_column :custom_fields, :show_filter, :boolean, after: :sort_priority, default: false, null: false
```

Comment:
I have a feeling that show_filter as name doesn't communicate the idea clearly. How about acts_as_filter or search_filter ? My thinking is that show_filter is a view level idea but there's bigger idea behind which is that some categories act as search filter and others act as additional listing information only.